src/components/TaskList.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { fetchData, sortDateTime } from "./../utils/utils";
+import AddTaskButton from "./../components/AddTaskButton";
+import {
+	MainCont,
+	TaskBox,
+	DateText,
+	Description,
+	TaskTitle,
+	TaskCont,
+	CheckBox,
+	Icon,
+} from "../styled-components/reusables";
+
+import tick from "./../images/tick.svg";
+
+export default function TaskList({ url }) {
+	const [tasks, setTasks] = useState(null);
+
+	useEffect(() => {
+		resolveTaskData();
+	}, []);
+
+	function resolveTaskData() {
+		fetchData(`${url}`)
+			.then((result) => setTasks(result))
+			.catch((error) => error);
+	}
+
+	return (
+		<>
+			<AddTaskButton />
+			<MainCont width="80vw" paddingTop="5rem">
+				{!(tasks instanceof TypeError) && tasks != null ? (
```

Comment:
This is interesting - why `TypeError` in particular?

Reply:
Because it was returning an error instead of null so my clause wasn't working which meant it kept trying to run Map of an error object.